modules/updating-control-plane-only-update.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/control-plane-only-update.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="updating-control-plane-only-update_{context}"]
+= Control Plane Only update
```

Comment:
If you're creating procedural content then the titles need to have a gerund in there to indicate that the content is procedural, as per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-headings#wording). In this case, I think you can just add "performing" to the beginning:
```suggestion
= Performing a Control Plane Only update
```